interface/globals.php

```diff
@@ -134,7 +134,9 @@
     // since this is user provided content we need to escape the value but we use htmlspecialchars instead
     // of text() as our helper functions are loaded in later on in this file.
     if (empty($tmp) || preg_match('/[^A-Za-z0-9\\-.]/', $tmp)) {
-        die("Site ID '". htmlspecialchars($tmp, ENT_NOQUOTES) . "' contains invalid characters.");
+	echo "Invalid URL";
```

Comment:
Isn't REST requester expecting json response?

Reply:
request has already been made. this is after and will die..

Reply:
I know request has already been made.  I also know echo before die is going to generate output that a browser client will see. 

The comment was specifically about making sure that response as seen by REST requesters are properly formatted per specifications.

Reply:
which brings us full circle to my original comment. The validation should be made in api which I will address soon.